server/audit_math/bravo.py

```diff
@@ -16,6 +17,9 @@
 from .sampler_contest import Contest
 from .ballot_polling_types import SampleSizeOption
 
+# Treat n/0 as Infinity instead of raising an error
+decimal.getcontext().traps[decimal.DivisionByZero] = False
```

Comment:
@umbernhard curious what you think about this approach vs handling the specific divide-by-zero case. I generally would lean towards failing loudly, but it seems more useful to have division be well-defined across the integers for our math. If you like this approach, we could enable it for other audit methods as well.